src/geojson/geojson_intf.ml

```diff
@@ -154,8 +154,8 @@ module type Geometry = sig
     val rings : t -> LineString.t array
     (** [rings t] returns the linear rings contained in [t] (a Polygon object) *)
 
-    val interior_ring : t -> LineString.t
-    val exterior_rings : t -> LineString.t array
+    val exterior_ring : t -> LineString.t
```

Comment:
Could we add some of documentation whilst we're making these changes? Just something based off of the description in the RFC should suffice :))